tensorboard/webapp/metrics/views/card_renderer/data_download_dialog_component.scss

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+:host {
+  width: 400px;
+}
+
+h2 {
+  align-items: baseline;
+  display: flex;
+  font-size: 1.25em;
+
+  .tag-name {
+    display: inline-block;
+    font-size: monospace;
+    max-width: 20ch;
+    overflow: hidden;
```

Comment:
1) There is currently no overflow because the width of your dialog is as wide as your longest element.

2) If you did decide on a fixed dialog width you have other flow problems you'd need to address:

![image](https://user-images.githubusercontent.com/17152369/112160099-38ed7580-8bc0-11eb-930a-86e64808cbee.png)

Reply:
Thanks for checking that. I've improved the narrow content styling.

![image](https://user-images.githubusercontent.com/2547313/112196496-53c3e800-8bc8-11eb-85ad-cf965e6c4656.png)